src/server/game/Maps/Map.h

```diff
@@ -430,7 +430,7 @@ class Map : public GridRefMgr<MapGridType>
     void UpdatePlayerZoneStats(uint32 oldZone, uint32 newZone);
     [[nodiscard]] uint32 ApplyDynamicModeRespawnScaling(WorldObject const* obj, uint32 respawnDelay) const;
 
-    TaskScheduler _creatureRespawnScheduler;
+    EventProcessor m_Events;
```

Comment:
```suggestion
    EventProcessor Events;
```
according to AC codestyle for **public** members, private member would be `_events`. should this even be public at all

Reply:
I would say that anything that is beeing touched in a pr should comply with the codestyle

Reply:
> I would say that anything that is beeing touched in a pr should comply with the codestyle

ok